src/Randomizer.App/Randomizer.App.csproj

```diff
@@ -5,7 +5,7 @@
     <TargetFramework>net5.0-windows</TargetFramework>
     <UseWPF>true</UseWPF>
     <ApplicationIcon>chozo20.ico</ApplicationIcon>
-    <Version>3.4.0</Version>
+    <Version>3.4.1</Version>
```

Comment:
Should the iss file not be updated as well?

Reply:
Yeah, I always forget until I make the setup itself.